tests/Elm/Parser/CaseExpressionTests.elm

```diff
@@ -62,34 +62,6 @@ True -> 1"""
                                 }
                             )
                         )
-        , test "case expression with trailing whitespace" <|
-            \() ->
-                """case f of
-  True -> 1
-  False -> 2
-
-"""
```

Comment:
Can you clarify why this test (and some similar ones) were removed?
Is it because these tests are hard to get to green, and that they should be done at a different level (like tests for declarations)?

I'm slightly worried that we're removing tests without replacing them.

Reply:
Yes, exactly, I think the expression parser shouldn't have to handle any more input once the expression is parsed.

But yes, I see your point, I will move those tests elsewhere rather than deleting them.